packages/leap/lib/src/leap_map.dart

```diff
@@ -120,6 +120,8 @@ class LeapMap extends PositionComponent with HasGameRef<LeapGame> {
       prefix: prefix,
       bundle: bundle,
       images: images,
```

Comment:
I wonder if `prefix`, `bundle`, and `images` should also be moved to `LeapTiledOptions`

Reply:
yeah, that might be a good idea! Those are only used while loading the tiled map iirc.

Should we merge this, then open an issue to track this other change?

Reply:
Sounds good! I mostly just want to keep the params from getting crazy here

Do you have permissions to merge? Feel free if so, not sure how if I need to add you or what

Reply:
@kurtome write permission in the repo is needed for me to merge the PR, I've been doing all these PRs from my fork :D